app/assets/javascripts/autocomplete.coffee

```diff
@@ -0,0 +1,54 @@
+delay = do ->
+  timer = 0
+  (callback, ms) ->
+    clearTimeout timer
+    timer = setTimeout(callback, ms)
+    return
+
+update_textfield = (list_element) ->
+  $(list_element).removeClass('suggestion-focused')
+  $('.js-autocomplete-textfield').val($(list_element).find('.res-name')[0].innerHTML)
```

Comment:
What do you think about modifying the form to submit `repository_id` when using autocompletion instead of sending `name/owner`?

I know it's an unlikely case, but it's possible that a user could rename the repository in between the time of choosing the item from the list and submitting the page. By using `repository_id` we avoid that